pkg/election/lease.go

```diff
@@ -80,7 +80,11 @@ func (l *lease) Close() error {
 	// Try to revoke lease to make subsequent elections faster.
 	ctx, cancel := context.WithTimeout(l.client.Ctx(), revokeLeaseTimeout)
 	defer cancel()
-	l.lease.Revoke(ctx, l.ID)
+	var leaseID clientv3.LeaseID
+	if l.ID.Load() != nil {
+		leaseID = l.ID.Load().(clientv3.LeaseID)
+	}
```

Comment:
if l.ID is nil, what will happen? Do we need to add some tests about it?

Reply:
if l.ID.Load() is nil, leaseID is initialized with the default value. Since clientv3.LeaseID is int64, so the initialized value is 0, which keeps exact the same as the existing logic.

Reply:
ok, I see